source/versioning.rst

```diff
@@ -0,0 +1,104 @@
+.. _versioning-conventions:
+
+Versioning Conventions
+======================
+
+One of the explicit goals of ASDF is to be as future proof as
+possible.  This involves being able to add features as needed while
+still allowing older libraries that may not understand those new
+features to reasonably make sense of the rest of the file.
+
+The ASDF standard includes three categories of versions, all of which
+may advance independently of one another.
+
+- **Standard version**: The version of the standard as a whole.  This
+  version provides a convenient handle to refer to a particular
+  snapshot of the ASDF standard at a given time.  This allows
+  libraries to advertise support for "ASDF standard version X.Y.Z".
+
+- **File format version**: Refers to the version of the blocking
+  scheme and other details of the low-level file layout.  This is the
+  number that appears on the ``#ASDF`` header line at the start of
+  every ASDF file and is essential to correctly interpreting the
+  various parts of an ASDF file.
+
+- **Schema versions**: Each schema for a particular YAML tag is
+  individually versioned.  We encourage schemas provided by third
+  parties (i.e. not in the ASDF specification itself) to be versioned
+  as well.
```

Comment:
(Just a suggestion--I think it clarifies the question of "Why should I use a version on my schema?"  I think it's obvious, but I would still anticipate the question.)